src/App.css

```diff
@@ -17,6 +17,20 @@ body {
   position: relative;
 }
 
```

Comment:
|Zeplin's Design| @kifle23 Design |
| --- | --- |
| ![image](https://github.com/kifle23/Bookstore/assets/93039359/fe8a57dd-14f2-4f9b-9744-a6472255f25a) | ![image](https://github.com/kifle23/Bookstore/assets/93039359/f6ca2b59-475c-4806-abd5-ee9375e79cbc) |

- [ ] It would be nice if you follow the project requirements and make your grey border width not more than the contents as you can see in the images above 👍 to make your project more professional 😊